### PR TITLE
Subpages/game

### PR DESCRIPTION
**Pull request by R4fau**

### CODE REVIEW

File: src/Player/__tests__/Player.spec.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Player } from "../player"
```
Comment (adam-kostuch) [inline src/Player/__tests__/Player.spec.ts]: There's no need to  import anything, when you don't use it. Remove it and add, after creating tests.

File: src/Player/__tests__/Player.spec.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+import { Node } from "../../node"
+import { Settings } from "../../settings"
+import { Team } from "src/team"
+
+describe("Player", () => {
+  test("it should have certain properties", () => {
+    // actual test
```
Comment (adam-kostuch) [inline src/Player/__tests__/Player.spec.ts]: Remove comment

File: src/Player/player.ts [hunk 3]
```diff
@@ -23,8 +23,8 @@ export class Player extends Entity {
 
     // validate if player can move there (walls) - TODO
```
Comment (adam-kostuch) [inline src/Player/player.ts]: We don't need comments anymore.

File: src/Player/player.ts [hunk 4]
```diff
@@ -23,8 +23,8 @@ export class Player extends Entity {
...
     this._currentNodeIdx = currentPos + x + y * dim
-    let blockX = currX + x
-    let blockY = currY + y
+    const blockX = currX + x
+    const blockY = currY + y
     // without ground
```
Comment (adam-kostuch) [inline src/Player/player.ts]: Same here

File: src/game/components/input/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/game/components/input/index.js]: Still js file

File: src/game/components/input/input.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/game/components/input/input.js]: JS one

File: src/node/__tests__/index.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/node/__tests__/index.js]: .js

File: src/node/__tests__/node.mock.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/node/__tests__/node.mock.js]: Same

File: src/node/components/draw/draw.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/node/components/draw/draw.js]: Here

File: src/node/components/draw/index.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/node/components/draw/index.js]: Here

File: src/node/components/index.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
+"use strict"
```
Comment (adam-kostuch) [inline src/node/components/index.js]: Here

**Review by adam-kostuch — CHANGES_REQUESTED** [review]

File: src/Player/__tests__/Player.spec.ts (hunk 2 above)
Comment (R4fau) [reply src/Player/__tests__/Player.spec.ts]: ok

File: src/game/components/input/index.js (hunk 5 above)
Comment (R4fau) [reply src/game/components/input/index.js]: deleted

**Review by adam-kostuch — APPROVED** [review]

**Pull request merged**